### PR TITLE
Fix QA issues in qca-swiss-army-knife packages

### PR DESCRIPTION
**Pull request by lumag**
Fix following QA issues in qca-swiss-army-knife packages:

```
ERROR: QA Issue: /usr/bin/ath10k-bdencoder contained in package qca-swiss-army-knife requires /usr/bin/python3, but no providers found in RDEPENDS_qca-swiss-army-knife? [file-rdeps]
ERROR: QA Issue: /usr/bin/read-powers contained in package qca-swiss-army-knife requires /usr/bin/python, but no providers found in RDEPENDS_qca-swiss-army-knife? [file-rdeps]
ERROR: QA Issue: /usr/bin/irq-watch contained in package qca-swiss-army-knife requires /usr/bin/perl, but no providers found in RDEPENDS_qca-swiss-army-knife? [file-rdeps]
```

### CODE REVIEW

**Review by kraj — APPROVED** [review]
Lgtm

File: recipes-devtools/qca-swiss-army-knife/qca-swiss-army-knife/0002-ath10k-fwencoder-port-to-python3.patch [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+ tools/scripts/ath10k/ath10k-fwencoder | 70 +++++++++++++--------------
+ 1 file changed, 35 insertions(+), 35 deletions(-)
+
+diff --git a/tools/scripts/ath10k/ath10k-fwencoder b/tools/scripts/ath10k/ath10k-fwencoder
+index 6934a694e832..bbbe0344f014 100755
+--- a/tools/scripts/ath10k/ath10k-fwencoder
++++ b/tools/scripts/ath10k/ath10k-fwencoder
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python
++#!/usr/bin/python3
```
Comment (kraj) [inline recipes-devtools/qca-swiss-army-knife/qca-swiss-army-knife/0002-ath10k-fwencoder-port-to-python3.patch:17]: It’s better to use usr/bin/env python3 so path is not hard coded

Comment (lumag) [reply recipes-devtools/qca-swiss-army-knife/qca-swiss-army-knife/0002-ath10k-fwencoder-port-to-python3.patch:17]: Done

**Review by kraj — APPROVED** [review]
thanks lgtm

**Pull request merged**